charts/platform-system/templates/flux/kyverno-policies/helm-release-kyverno-policies.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if and (and .Values.apps.kyverno_policies.enabled .Values.apps.kyverno.enabled) (eq .Values.solution.name "flux") }}
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kyverno-policies
+  namespace: kyverno
```

Comment:
Make sure to use the `Values` w/ default, maybe for `name` too. Wondering if we should prefix all the things too.

Reply:
I need clarification on what you're wanting to happen here wrt 'default'. I think prefixing is going to look really messy. just my two cents.

Reply:
Is there not a way to spread operator if
```{{ defaultValues := .Values.kyerno }}
kind: HelmRelease
metadata:
  name: kyverno-policies
  namespace: kyverno
spec: {{ toYaml . }}```

Reply:
> I need clarification on what you're wanting to happen here wrt 'default'. I think prefixing is going to look really messy. just my two cents.

Fair. Similar to the `cert-manager` comment, we should probably put the `HelmRelease` in `flux-system` and `targetNamespace` from values with a default. Though here the `targetNamespace` is probably `apps.kyverno.namespace` unless a namespace is specified for the policies that differs. Agree we skip prefix. There shouldn't be a conflict on any of these.